### DISCUSSION TITLE
Unable to send emails using AWS SES

### DISCUSSION THREAD

**Discussion by g1sbi** [pocketbase/pocketbase#2803] [Q&A] [answered]
I set up the SMTP settings for my SES identity in Pocketbase. I'm using StartTLS on port 25. I made sure to open the port 25 on my VPS. If I send a test email directly from the aws console to the same email address it works, whereas in Pocketbase I get a timeout. Are there any other settings I should be aware of that could block the email?

**Comment by g1sbi** [comment]
I should note that my SES is still in a sandbox, but I added my email address to the verified ones for testing

**Reply by ganigeorgiev** [reply, marked as answer]
I'm not sure what could be the reason for the timeout but you can try starting PocketBase with `--debug` and check if there is any additional error output in the console.

Also note that a lot of VPS providers may block outgoing common mail ports like 25, 465, 587 and in this case you have to contact their support to enable the port you want to use (Hetzner for example has 1 month block for 25 and 465 but 587 is allowed).

For SMTP test purposes I often use the free plan of [mailtrap.io](https://mailtrap.io/) (_they also have a convenient email preview and "spam score" based on common Spamassassin filters_).

**Reply by g1sbi** [reply]
Correct, apparently Digital Ocean blocks the port by default. Thank you.
